docs/ref/templates/api.txt

```diff
@@ -497,6 +497,17 @@ the stack instead of an empty one.
     >>> c['foo']
     'first level'
 
+Like ``push()``, you can use ``update()`` as a context manager to ensure a
+matching ``pop()`` is called.
+
+    >>> c = Context()
+    >>> c['foo'] = 'first level'
+    >>> with c.update({'foo': 'second level'}):
+    >>>     c['foo']
+    'second level'
+    >>> c['foo']
+    'first level'
+
```

Comment:
need a `.. versionadded:: 1.9` annotation below this section saying "The ability to use update() as a context manager was added." and a mention in the 1.9 release notes